jupyter/Dockerfile.centos

```diff
@@ -1,7 +1,14 @@
 FROM centos:8
 
 RUN yum update -y && yum install -y \
+    which \
```

Comment:
should we add the nice to have deps in this image like we did in the others?
- git, bash, vim, curl, htop, ssh, etc.

Reply:
Done